### DISCUSSION TITLE
OSPOlogy Follow-up: How Academic OSPOs Are Amplifying Research Impact

### DISCUSSION THREAD

**Discussion by anajsana** [todogroup/ospology#79] [OSPO Stories]
Discuss with the community topics related to the upcoming OSPOlogy meeting:

📅 April, 6th [How Academic OSPOs Are Amplifying Research Impact](https://community.linuxfoundation.org/events/details/lfhq-todo-group-presents-how-academic-ospos-are-amplifying-research-impact/)

In this open space, people can:

* Ask questions prior to the meeting starts
* Share notes from the conversation
*Make proposals for new initiatives that can help the OSPO Ecosystem
* Add useful links and resources related to the topic
* Connect with mentors involved in OSPOs and/or Open Source Management

**Comment by anajsana** [comment]
Slides for the meeting are available [here](https://docs.google.com/presentation/d/1rXMmCFElVzcEeqNUA8Jf_2X_Dl8Kpb_11lcjGHXGXRk/edit?usp=sharing)

Also, people can download the pdf version:
[8OSPOlogy meeting 8.pdf](https://github.com/todogroup/ospology/files/8427893/8OSPOlogy.meeting.8.pdf)

**Reply by souravcipher** [reply]
Thank you @anajsana!

**Reply by carlosmalt** [reply]
Our presentation includes two videos -- they are both accessible even from the pdf version. The second video is still a draft and we expect to post the final version next week. Please don't distribute the draft version too widely.

**Reply by anajsana** [reply]
@carlosmalt you can share the youtube video link here too 👍

**Comment by souravcipher** [comment]
Q. What were the most difficulties building OSPO in an Academia environment and how did you overcome it?
Q. What are your top 3 missions and how do you plan to achieve it?
Q. Are there any other academia building/have OSPO?

**Reply by carlosmalt** [reply]
@souravrhythm: 
Q1: One was overcoming the dominance of industry "university relations" which are primarily interested in recruiting students and don't necessarily understand the value of encouraging open source practices and communities at universities. Before we knew about the OSPO network, we had a hard time to get past "university relations" to the "open source strategy" folks.

**Reply by anajsana** [reply]
Q3: You can find all the "OSPO adopters"(including academia) in the OSPO landscape: https://landscape.todogroup.org/

**Reply by carlosmalt** [reply]
Q2: Increase open source literacy for everyone, especially historically excluded communities, track the value of open source, and achieve broad industry engagement. We have a number of programs designed to achieve these missions. Perhaps the most important one is our Open Source Research Experience that creates a market of project ideas, mentors, students, and sponsors and leverages GSoC and other outreach efforts.

**Comment by winterrocks** [comment]
Q: What are the main differences you see between an university OSPO vs. for-profit company OSPO?
Q: What are the main similarities you see between an university OSPO vs. for-profit company OSPO?

**Comment by anajsana** [comment]
Question from @souravrhythm 

Whats your suggestions for other academia who are interested in starting an OSPO and what can we learn more Academia OSPO - which can be implemented in OSPO in For profit companies?

**Reply by anajsana** [reply]
Also, "How much time(months/year) it took for you to setup the OSPO and launch it? and could you please tell us about your OSPO team structure like how many members do you have what are the members roles?"

**Comment by anajsana** [comment]
question from @itprofjacobs 

Are you acting as a SC hub for GSOC or helping each mentor list their project individually?
